.circleci/integration-tests/master_dag.py

```diff
@@ -132,9 +132,7 @@ def prepare_dag_dependency(task_info, execution_time):
         {"gcs_sensor_dag": "example_async_gcs_sensors"},
         {"big_query_sensor_dag": "example_bigquery_sensors"},
         {"dataproc_dag": "example_gcp_dataproc"},
-        # GkeStartPod operator do not work on astro-cloud
-        # https://github.com/astronomer/astronomer-providers/issues/443
-        # {"kubernetes_engine_dag": "example_google_kubernetes_engine"},
+        {"kubernetes_engine_dag": "example_google_kubernetes_engine"},
```

Comment:
what was the fix @phanikumv or its working without any fixes?

Reply:
It works without any fixes(the OSS airflow fix worked for this)

Reply:
that great 🚀